BruTile/Web/ArcGisTileRequest.cs

```diff
@@ -9,15 +9,15 @@
 
 namespace BruTile.Web;
 
-public class ArcGisTileRequest(Uri baseUrl, string format, Dictionary<string, string> customParameters) : IRequest
+public class ArcGisTileRequest(Uri baseUrl, string format, Dictionary<string, string> customParameters) : IUrlBuilder

```

Comment:
Use IUrlBuilder instead of IRequest. I associate IRequest with the http call itself. I think I will rename the implementations as well.